pages/atoms/cart-button/cart-button.tsx

```diff
@@ -0,0 +1,22 @@
+import Image from "next/image";
+import { NavigationData } from "../../../data";
+import style from "../cart-button/cart-button.module.scss";
+
+export default function CartButton(props: NavigationData) {
```

Comment:
define new component prop type and use it, could undo some change for NavigationData (making properties optional)